frontend/public/components/utils/units.js

```diff
@@ -175,16 +184,21 @@ export const humanizeCpuCores = v => {
   const value = v < 1 ? round(v*1000) : v;
   const unit = v < 1 ? 'm' : '';
   return {
-    string: `${value}${unit}`,
+    string: `${formatValue(value)}${unit}`,
     unit,
     value,
   };
 };
-export const humanizePercentage = value => ({
-  string: formatPercentage(value/100),
-  unit: '%',
-  value,
-});
+export const humanizePercentage = value => {
+  if (!isFinite(value)) {
+    value = 0;
```

Comment:
I would expect this to return the empty string to differentiate between 0 and no value